analyzers/src/SonarAnalyzer.CSharp/Rules/ParametersCorrectOrder.cs

```diff
@@ -35,6 +35,16 @@ public sealed class ParametersCorrectOrder : ParametersCorrectOrderBase<Argument
     {
         private static readonly DiagnosticDescriptor rule =
             DiagnosticDescriptorBuilder.GetDescriptor(DiagnosticId, MessageFormat, RspecStrings.ResourceManager);
+
+        private static SyntaxToken? GetExpressionSyntaxIdentifier(ExpressionSyntax expressionSyntax) =>
```

Comment:
Private methods should be at the end of the class by our https://github.com/SonarSource/sonar-dotnet/blob/master/docs/coding-style.md